features/in_onderzoek.feature

```diff
@@ -884,34 +1043,77 @@ Functionaliteit: in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                        | waarde    |
       | burgerservicenummer         | 555550001 |
-      En de persoon met burgerservicenummer '555550001' heeft de volgende 'naam' gegevens
-      | naam                          | waarde                  |
-      | voornamen (02.10)             | Piet                    |
-      | geslachtsnaam (02.40)         | Groenen                 |
-      | aanduidingNaamgebruik (61.10) | <aanduidingNaamgebruik> |
-      En de persoon heeft een partner met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 050200   |
-      | datumIngangOnderzoek          | 20220307 |
+      | geslachtsaanduiding (04.10) | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde                  |
+      | voornamen (02.10)              | Piet                    |
+      | geslachtsnaam (02.40)          | Groenen                 |
+      | aanduiding naamgebruik (61.10) | <aanduidingNaamgebruik> |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                     | waarde    |
+      | burgerservicenummer                      | 555550002 |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde>  |
+      | datum ingang onderzoek (83.20)           | 20220307  |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | geslachtsnaam (02.40) | Geel   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                       |
+      | type                | RaadpleegMetBurgerservicenummer              |
+      | burgerservicenummer | 555550001                                    |
+      | fields              | aanhef,aanschrijfwijze,gebruikInLopendeTekst |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
```

Comment:
moet hier het woord "alleen" niet worden toegevoegd om te controleren dat inOnderzoek niet is opgenomen?